### DISCUSSION TITLE
Can i just drop a connection/request?

### DISCUSSION THREAD

**Discussion by pixelpaulaus** [adonisjs/core#4272] [Help]
Hi All,
I am uing adonis for an API and it works great. But I get a LOT of bots and people probing the API to try hack/break it.
I can send a 404 error or something, but they just keep trying.
Is there a way I can just drop the request and send nothing, so they stay waiting for a response but never get one?

**Comment by RomainLanz** [comment]
Hey @pixelpaulaus! 👋🏻 

I don't think that will do anything. Because they probably have a timeout after 10s, so it will only help a little.

I would recommend using the [rate limiter](https://docs.adonisjs.com/guides/rate-limiting) package.
You could also activate the "Under Attack" option on Cloudflare.
